### ISSUE TITLE
traing loss converge problem

### ISSUE THREAD

**Issue by griffintin** [melfm/avod-ssd#3]
@melfm 
Hi, Sorry to bother you.
I have tried to train/validate avod_ssd using provided data. And the loss is like following. 

![20180426_avod_ssd_loss](https://user-images.githubusercontent.com/31429693/39286536-79cf09f8-4959-11e8-8cec-04045f4e668e.png)

Although by iteratively evaluate the checkpoints, best AP at iter=140000 is like following, which is very close to yours.
83.17% | 73.49% | 67.58%

Why loss is so unstable? how could i improve this?
By the way, the original avod_ssd_cars_example.config and training data are used, no change was made.
